Pod/API/PublicRemoteMethods/consoleCommand.swift

```diff
@@ -0,0 +1,85 @@
+//
+//  consoleCommand.swift
```

Comment:
Maybe.  The convention I'd been using (which may well be a mistake) was that the public remote method file names weren't capitalized.